src/components/base/Tooltip/index.tsx

```diff
@@ -0,0 +1,30 @@
+import type { ReactElement } from 'react';
+import { StyledTooltipContent, StyledTooltipWrapper } from './styled';
+import type { TooltipProps } from './types';
+import { Text } from '@base';
+
+const Tooltip = ({
+  children,
+  tooltipMessage = '툴팁메시지',
+  direction = 'top',
+  tooltipSize = 'md',
+  tooltipMessageSize = 'md',
+  ...props
+}: TooltipProps): ReactElement => {
+  return (
+    <StyledTooltipWrapper className='tooltipcontainer' {...props}>
+      {children}
+      <StyledTooltipContent
+        className='tooltiptext'
+        direction={direction}
+        size={tooltipSize}
+      >
+        <Text color={'BASIC_WHITE'} size={tooltipMessageSize}>
```

Comment:
```suggestion
        <Text color='BASIC_WHITE' size={tooltipMessageSize}>
```
전혀중요한건 아니지만, props에 string값으로 전달할때는 중괄호 {} 빼고 작성해도 되더라구요. 개인적으로 이게 더 이쁘다고 생각해서 일단 남겨만 둡니다~